src/UISettings.hpp

```diff
@@ -52,8 +52,17 @@ struct UISettings {
   /** Override OS dpi settings */
   unsigned custom_dpi;
 
-  /** Show ThermalAssistant if circling */
-  bool enable_thermal_assistant_gauge;
+  /** Show ThermalAssistant if circling (obsolete - use thermal_assistant_position)*/
+  bool enable_thermal_assistant_gauge_obsolete;
```

Comment:
Nobody uses this variable, so what's the point of storing it?

Reply:
Used in UIProfile.cpp to to migrate the old setting, as required in the previous PR

Reply:
Have moved it to a local variable in UIProfile and rebased.